routes/web.php

```diff
@@ -83,6 +83,40 @@
             });
         });
 
+        //TODO: remove this before merging into develop
+//        Route::prefix(__('route.products'))->name('products.')->group(function () {
+//            Route::get('/', [ProductController::class, 'index'])
+//                ->name('index');
+//
+//            Route::prefix(__('route.create'))->name('create.')->group(function () {
+//                Route::get('/', [ProductController::class, 'add'])
+//                    ->name('index');
+//
+//                Route::post('store', [ProductController::class, 'store'])
+//                    ->name('store');
+//            });
+//
+//            Route::prefix(__('route.edit'))->name('edit.')->group(function () {
+//                Route::get('/{id}', [ProductController::class, 'edit'])
+//                    ->name('index');
+//
+//                Route::put('store/{id}', [ProductController::class, 'update'])
+//                    ->name('store');
+//            });
+//        });
+
+        // Manage Stocks
+        Route::prefix(__('navbar.manage_stocks'))->group(function () {
```

Comment:
```suggestion
        Route::prefix(__('route.stocks'))->name('stocks.')->group(function () {
```
En vervolgens in route.php: `'stocks' => 'voorraad'`